archinfo/arch_soot.py

```diff
@@ -1,12 +1,17 @@
 
 from .arch import Arch, register_arch, Endness
+import re
+
+import logging
+l = logging.getLogger('archinfo.arch_soot')
 
```

Comment:
The import sequence:

1. Python built-in packages.
1. Third-party libraries.
1. Other angr libraries.
1. Modules from the current module (archinfo).

Make sure to leave a blank line between each group of imports.

So ideally it should look like the following:

```
import logging
import re

from .arch import Arch, xxx


l = logging.xxx
```